### PR TITLE
Build clinic database schema

### PR DESCRIPTION
**Pull request by ivangonzalez224**
It is our Clinic Database and we added the following features:

- Created the patients, medical_histories, invoices, treatments, and invoice_items tables.
- Added foreign keys in the medical_histories, invoices, and invoice_items tables.
- Created the medical_histories_treatments many-to-many join table.
- Added foreign keys in the medical_histories_treatments table.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+-- Create the medical_histories_treatments many-to-many join table
+CREATE TABLE medical_histories_treatments (
+   ID INT GENERATED ALWAYS AS IDENTITY,
+   MEDICAL_HISTORIES_ID  INT,
+   TREATMENT_ID INT,
+   CONSTRAINT FK_MEDICAL_HISTORIES_ID FOREIGN KEY(MEDICAL_HISTORIES_ID) REFERENCES medical_histories(ID),
+   CONSTRAINT FK_TREATMENT_ID FOREIGN KEY(TREATMENT_ID) REFERENCES treatments(ID),
+   PRIMARY KEY(ID)
+);
```
Comment (ridwanediallo) [inline schema_based_on_diagram.sql]: - [ ] Great, job creating the tables based on the diagram. However, as requested please ensure that you add indexes to the foreign keys. The indexes help speed up query performance. 

### Requirement :point_down: 

Screenshot from 2023-09-26 21-49-49

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi team, 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done creating the tables based on the diagram. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (ivangonzalez224) [reply schema_based_on_diagram.sql]: Hi @ridwanediallo thanks for the review.
Changes done: FK indexes added.

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi team, 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- The tables have successfully been created based on the diagram. ✔
- The milestone requirements have been met with success. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**